.github/workflows/close_stale_issues.yaml

```diff
@@ -0,0 +1,21 @@
+name: Close stale issues
+on:
+  schedule:
+  - cron: '0 0 * * *'
+jobs:
+  close-stale-issues:
+    runs-on: ubuntu-latest
+    steps:
+    - name: close stale issues
+      uses: actions/stale@v3
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        #This message is requiered but never used.
+        stale-issue-message: "."
```

Comment:
What does this mean?

Reply:
By default the action will leave a message when marking the issue as stale and another-one when closing the issue. As we only want the message to be added when the issue is closed the "skip-stale-issue-message" parameter was set to true but the action wont work if the "stale-issue-message" doesn't have a value.

Reply:
@Astul-Betizagasti does empty string works in this case?

Reply:
Thanks for the updates